src/e2e/puppeteer/__tests__/color.ts

```diff
@@ -163,3 +164,30 @@ it('Empty <span> element will be removed after setting color to default.', async
   const result = await getEditingText()
   expect(result).toBe('Golden Retriever')
 })
+
+it('Set the superScript color of the text', async () => {
+  const importText = `
+    - a
+    - a
+    - hello world
+    - hello world
+    `
+  await paste(importText)
+
+  await waitForEditable('hello world')
+  await clickThought('hello world')
+  await setSelection(6, 11)
+  await click('[data-testid="toolbar-icon"][aria-label="Text Color"]')
+  await click('[aria-label="text color swatches"] [aria-label="red"]')
+
+  const supColor2 = await getSuperScriptColor()
+  expect(supColor2).toBe(null)
```

Comment:
This test could use a couple comments that explains the two assertions, one positive and one negative case. I'm okay having these two cases in a single test for efficiency, just needs a little explanation.